src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -939,6 +940,8 @@ int bus_socket_exec(sd_bus *b) {
                 if (rearrange_stdio(s[1], s[1], STDERR_FILENO) < 0)
```

Comment:
well, in real life not, but if any of the first three fds is closed when we enter this code this might not be the case... And this case is potentitally called from foreign environments, who knows (think NSS, where we already had problems with that!)